microsoft/testsuites/dpdk/dpdkutil.py

```diff
@@ -494,12 +423,12 @@ def verify_dpdk_build(
     log: Logger,
     variables: Dict[str, Any],
     pmd: str,
+    hugepage_size: HugePageSize,
     multiple_queues: bool = False,
-    gibibyte_hugepages: bool = False,
 ) -> DpdkTestResources:
     # setup and unwrap the resources for this test
     test_kit = initialize_node_resources(
```

Comment:
You now need to try/catch here to catch NotEnoughMemoryException and raise it as a SkippedException. You can encapsulate Exceptions within each other. Idk is this the exact syntax but something along the lines of:
```
except NotEnoughMemoryException as err:
    raise SkippedException(err)
 ```

Reply:
Like i mentioned in the previous reply, I have kept it intensionally like that so that the test actually fails with the reasoning

Reply:
same as above